### ISSUE TITLE
Add escape support

### ISSUE THREAD

**Issue by derdesign** [chjj/liquor#4]
Hello Chris,

Would it be too difficult to add escape support to the liquor engine ? Something like:

`#{{string}}`

The code above would insert the escaped string into the liquor template.

While using liquor, I've noticed I've had to use the same escape function everywhere, so I thought having built-in escaping of unsafe characters would be very helpful.

All the best,
Ernie.-

**Comment by chjj** [comment]
Yeah, I guess this should be expected of a templating engine. I'll see what I can do.
